src/components/organisms/Room/Room.tsx

```diff
@@ -155,18 +56,24 @@ const Room: React.FC<RoomProps> = ({
   }, [participants.length]);
 
   const meComponent = useMemo(() => {
-    return room && profile ? (
-      <div className={`participant-container ${participantContainerClassName}`}>
-        <LocalParticipant
-          key={room.localParticipant.sid}
-          participant={room.localParticipant}
-          profileData={profile}
-          profileDataId={room.localParticipant.identity}
+    return localParticipant && userWithId ? (
+      <div
+        className={`participant-container ${participantContainerClassName}`}
+        key={localParticipant.identity}
+      >
+        <VideoParticipant
+          participant={localParticipant}
+          participantUser={userWithId}
           defaultMute={defaultMute}
         />
       </div>
     ) : null;
```

Comment:
```suggestion
    return localParticipant && userWithId &&
      <div
        className={`participant-container ${participantContainerClassName}`}
        key={localParticipant.identity}
      >
        <VideoParticipant
          participant={localParticipant}
          participantUser={userWithId}
          defaultMute={defaultMute}
        />
      </div>
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/2369/commits/8fa06206d8a463463d85380f61fff2b4dd953adb